### PR TITLE
Cargo build now builds to /target/debug/

### PR DESCRIPTION
**Pull request by johnz133**
Changed the docs to reflect that

### CODE REVIEW

File: src/doc/trpl/hello-cargo.md [hunk 1]
```diff
@@ -78,16 +78,16 @@ Once you have this file in place, we should be ready to build! Try this:
...
 ```
 
 Bam! We build our project with `cargo build`, and run it with
-`./target/hello_world`. This hasn't bought us a whole lot over our simple use
+`./target/debug/hello_world`. This hasn't bought us a whole lot over our simple use
 of `rustc`, but think about the future: when our project has more than one
 file, we would need to call `rustc` more than once, and pass it a bunch of options to
 tell it to build everything together. With Cargo, as our project grows, we can
-just `cargo build` and it'll work the right way.
+just `cargo build` and it'll work the right way.
```
Comment (huonw) [inline src/doc/trpl/hello-cargo.md]: Accidental trailing space?

Comment (johnz133) [inline src/doc/trpl/hello-cargo.md]: Ah yes. Just fixed it!

**Comment by johnz133** [comment]
ah, yes. Just fixed it! (commit message: redundant*)

**Comment by steveklabnik** [comment]
:confetti_ball:

**Pull request merged**